components/engine_rocks/src/raw_util.rs

```diff
@@ -157,7 +157,7 @@ pub fn new_engine_opt(
         }
     }
     let cfds = cfs_v.into_iter().zip(cfs_opts_v).collect();
-    let mut db = DB::open_cf(db_opt, path, cfds).unwrap();
+    let mut db = DB::open_cf(db_opt, path, cfds)?;
```

Comment:
Is it safe to return error?

Reply:
Caller should do the right thing when getting the error, I think?

Reply:
I'm not sure if the error is handled properly by panicking.

Reply:
RocksDB returning error on open is a clean state. No need to panic actually.

Reply:
Then how to recover from the error?